src/Components/Form/AutoCompleteAsync.tsx

```diff
@@ -89,7 +89,10 @@ const AutoCompleteAsync = (props: Props) => {
               }
               onChange={({ target }) => setQuery(target.value)}
               onFocus={props.onFocus}
-              onBlur={props.onBlur}
+              onBlur={() => {
+                setQuery("");
+              }}
```

Comment:
So what happens when I pass an `onBlur` prop to this component? It'll not work, right?
This might break some desired behaviours of other existing usages.

Reply:
onblur prop will be a function right? I will just add passed prop as well in the arrow function?

Reply:
Yes!

Reply:
ok

Reply:
You can either remove the support of having `onBlur` from the props if it's not used anywhere, or you can ignore sonarcloud and still call the `onBlur` passed via the props.

Reply:
i have added the onblur that is passed from the props.